reference/index.rst

```diff
@@ -6,3 +6,4 @@ Reference
 
     content-types/index
     twig-extensions/index
+    components/document-manager/index.rst
```

Comment:
I don't like the location of this documentation. Ultimately some of this documentation should be in the *book*, and the rest (e.g. the Subscribers) should be in *reference*.